webofneeds/won-owner-webapp/src/main/webapp/config/details/price.js

```diff
@@ -157,7 +157,7 @@ export const price = {
   ],
   messageEnabled: true,
   parseToRDF: function({ value, identifier, contentUri }) {
-    if (!value || !value.amount || !value.currency) {
```

Comment:
maybe we should handle negative prices here (also?)

Reply:
Fixed this and a few other issues with 0 being evaluated as false.